databricks/koalas/series.py

```diff
@@ -548,6 +548,96 @@ def to_dict(self, into=dict):
         return validate_arguments_and_invoke_function(
             kseries.to_pandas(), self.to_dict, pd.Series.to_dict, args)
 
+    def to_latex(self, buf=None, columns=None, col_space=None, header=True, index=True,
+                 na_rep='NaN', formatters=None, float_format=None, sparsify=None, index_names=True,
+                 bold_rows=False, column_format=None, longtable=None, escape=None, encoding=None,
+                 decimal='.', multicolumn=None, multicolumn_format=None, multirow=None):
+        r"""
+        Render an object to a LaTeX tabular environment table.
```

Comment:
is the doc here identical to DataFrame's? If yes, maybe we can do something like `to_latex.__doc__ = DataFrame.to_latex.__doc__` to avoid duplicating everything?

Reply:
Great idea! Yes, they are the same.

Adding this in lieu of the docstring in series.py shortly.